.github/SECURITY.md

```diff
@@ -0,0 +1,74 @@
+# Security Policy
+
+We truly appreciate efforts to discover and disclose security issues responsibly!
+
+## Vulnerabilities
+
+If you'd like to report a security issue in the repositories of matter-labs organization, please proceed to our
+[Bug Bounty Program on Immunefi](https://era.zksync.io/docs/reference/troubleshooting/audit-bug-bounty.html#bug-bounty-program).
```

Comment:
The link is broken, you can find the correct one in the zksync-era repo (notion doc has already been updated)